src/borg/helpers.py

```diff
@@ -51,6 +51,15 @@ def Chunk(data, **meta):
     return _Chunk(meta, data)
 
 
+exit_code = EXIT_SUCCESS
```

Comment:
can you put a comment above this, explaining about it and archiver.exit_code?

the rest lgtm.